README.md

```diff
@@ -142,3 +142,15 @@ For development, install [poetry](https://python-poetry.org/docs/) and run `poet
 Install pre-commit with `pre-commit install` (don't forget you need to prepend all commands with `poetry run ...` if you did not run `poetry shell` first).
 
 Run tests by calling `pytest tests`.
+
+## Dataset and Benchmark Paper
```

Comment:
Please also add the section to the table of contents at the top (it's not created automatically..)

Reply:
Added in https://github.com/rossumai/docile/pull/55/commits/8231c230cc0ed7e303481c107f6b94cadef9babf .